_posts/2018-10-22-story-about-one-small-library.md

```diff
@@ -0,0 +1,71 @@
+---
+layout:     post
+title:      Story about one small library
+date:       2018-10-15
+summary:    Ethereumex evolution
+categories: ethereum
+---
+
+![babel](https://i.imgur.com/0lgyBVW.jpg)
+
+### Introduction
+
+A couple of years ago I got interested in Ethereum. My favourite programming language at the time was Elixir (by the way, Elixir is still my favourite language). My first contribution to the Ethereum community was [Ethereumex](https://github.com/exthereum/ethereumex) - Elixir JSON-RPC client for the Ethereum.
+
+In this post, I'll describe its evolution. I have got a couple of other open source projects that I wrote or was a part of the team. But I want to describe the evolution of Ethereumex because this small library is a good example of a hobby project that was written on the weekend and then got relatively popular in the community. I won't go into too much detail about the technical details of Ethereumex as I want this post to be about the open source, not about technology that it implements.
+
+### History
+
+From the first days of the project, I tried to follow community guides of good open source projects. One of the things I did for this is I added `CHANGELOG.md` file to easier track changes between versions. I'll use this file to describe the development of Ethereumex.
```

Comment:
What do you think about "I did for this _was to add a_ CHANGELOG.md file to _more easily_ track changes between versions"